Source/Components/Touchpoints/LiveImages/MLBusinessLiveImagesView.swift

```diff
@@ -94,16 +95,13 @@ extension MLBusinessLiveImagesView: MLBusinessLiveImagesHelper {
     }
     
     func pause() {
-        changeState(to: .stoped)
-        transitionView()
+        viewModel.prepareForStoping(state: liveImageState)
```

Comment:
CREO que la forma correcta de decirlo en inglés sería así. De todas formas no es algo crítico. Fijate si es así y si tenemos que meter algún otro cambio lo podemos modificar de ser necesario.
```suggestion
        viewModel.prepareToStop(state: liveImageState)
```